### DISCUSSION TITLE
相关视频栏目中竖屏封面的视频展示错误

### DISCUSSION THREAD

**Discussion by PRO-2684** [ywmoyue/biliuwp-lite#950] [General]
### Bug 描述

![bili](https://github.com/ywmoyue/biliuwp-lite/assets/54608551/8e2831fc-d628-4fc9-b974-ceebd1c4a05c)

### 操作步骤

```plain text
刷视频，看到有推荐竖屏封面的视频时即可复现
```

### 期望结果

与其它视频占用相同高度

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows11 23H2

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
是指首页的推荐Tab还是视频详情页的推荐Tab

**Comment by PRO-2684** [comment]
> 是指首页的推荐Tab还是视频详情页的推荐Tab

视频详情页

**Comment by ywmoyue** [comment]
这个问题我一直没刷到过，看了这个位置代码里是有设置最大高度的

**Comment by ywmoyue** [comment]
一直未复现，也没有其他人反馈，转discussion并设置待测试标签
